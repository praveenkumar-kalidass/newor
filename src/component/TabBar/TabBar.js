import React from 'react';
import { TouchableOpacity } from 'react-native';
import { useNavigation } from '@react-navigation/native';
import { Box, CircleIcon, Text } from 'native-base';
import FontAwesome from 'react-native-vector-icons/FontAwesome';

import CONSTANT from 'constant';
import ROUTE from 'constant/route';
import AppButton from 'component/AppButton';
import useTheme from 'theme/useTheme';
import Translation from 'translation/Translation';
import {
  TabContainer,
  TabNavigations,
  TabBackgroundContainer,
  TabCenterIconContainer,
  TabCenterIconBackground,
} from './TabBar.style';

const TabBar = () => {
  const navigation = useNavigation();
  const theme = useTheme();

  return (
    <TabContainer>
      <TabNavigations>
        <TabCenterIconContainer>
          <TabCenterIconBackground />
        </TabCenterIconContainer>
        <TabBackgroundContainer />
        <Box flex={1} mb={1} ml={4} alignItems="center" opacity={0}>
          <TouchableOpacity>
            <CircleIcon size={10} />
          </TouchableOpacity>
          <Text>Liabilities</Text>
        </Box>
        <Box flex={1} mb={1} alignItems="center">
          <AppButton
            variant="primary"
            rounded="full"
            onPress={() => navigation.navigate(ROUTE.DASHBOARD)}
            size={16}
            _text={{ fontSize: 36 }}
          >
            {CONSTANT.APP_LITERAL.RUPEE_SYMBOL}
          </AppButton>
          <Text>&nbsp;</Text>
        </Box>
        <Box flex={1} mb={2} mr={4}>
          <TouchableOpacity onPress={() => navigation.navigate(ROUTE.ASSET)}>
            <Box alignItems="center">
              <Box flexDirection="row" alignItems="flex-end" p={1} borderRadius={5}>
                <FontAwesome color={theme.color.BACKGROUND_0} name="home" size={30} />
                <FontAwesome color={theme.color.BACKGROUND_0} name="car" size={20} />
                <FontAwesome color={theme.color.BACKGROUND_0} name="rupee" size={30} />
              </Box>
              <Translation tkey="ASSETS" as={Text} bold fontSize={16} />
            </Box>
          </TouchableOpacity>
        </Box>
      </TabNavigations>
    </TabContainer>
  );
};

export default TabBar;
