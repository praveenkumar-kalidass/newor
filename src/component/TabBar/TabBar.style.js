import styled from 'styled-components/native';
import { Box, HStack } from 'native-base';

export const TabContainer = styled(Box).attrs(({ theme }) => ({
  bg: theme.color.PRIMARY_100,
  p: 6,
}))``;

export const TabNavigations = styled(HStack).attrs(() => ({
  bg: 'transparent',
  justifyContent: 'space-around',
  alignItems: 'flex-end',
}))``;

export const TabBackgroundContainer = styled(Box).attrs(({ theme }) => ({
  position: 'absolute',
  bg: theme.color.MODAL,
  w: '100%',
  p: 10,
  rounded: 10,
  elevation: 1,
  shadowColor: theme.color.BACKGROUND_0,
  shadowOffset: { width: 0, height: 1 },
  shadowOpacity: 0.8,
  shadowRadius: 1,
}))``;

export const TabCenterIconContainer = styled(Box).attrs(({ theme }) => ({
  alignItems: 'center',
  position: 'absolute',
  bg: 'transparent',
  w: '100%',
  elevation: 1,
  shadowColor: theme.color.BACKGROUND_0,
  shadowOffset: { width: 0, height: 1 },
  shadowOpacity: 0.8,
  shadowRadius: 1,
}))``;

export const TabCenterIconBackground = styled(Box).attrs(({ theme }) => ({
  bg: theme.color.MODAL,
  w: 24,
  h: 24,
  mb: 3,
  rounded: 'full',
}))``;
