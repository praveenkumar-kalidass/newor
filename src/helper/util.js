import CONSTANT from 'constant';

export const formatCurrency = (value) => {
  const formatter = new Intl.NumberFormat('hi-IN', { style: 'currency', currency: 'INR' });
  return formatter.format(value);
};

export const unformatCurrency = (value) => {
  const withoutSymbol = value.replace(CONSTANT.APP_LITERAL.RUPEE_SYMBOL, '');
  const currency = withoutSymbol.replace(/,/g, '');
  return parseFloat(currency);
};

export const formatUrlParams = (data) => Object.keys(data).map((key) => (
  `${encodeURIComponent(key)}=${encodeURIComponent(data[key])}`
)).join('&');
