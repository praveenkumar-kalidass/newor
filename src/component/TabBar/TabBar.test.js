import React from 'react';

import { render, fireEvent } from 'test/util';
import TabBar from './TabBar';

const mockNavigation = {
  navigate: jest.fn(),
};
jest.mock('@react-navigation/native', () => {
  const reactNavigation = jest.requireActual('@react-navigation/native');
  return {
    ...reactNavigation,
    useNavigation: () => mockNavigation,
  };
});

describe('AppModal', () => {
  it('should match snapshot', () => {
    const container = render(<TabBar />);

    expect(container).toMatchSnapshot();
  });

  it('should navigate to Asset screen', () => {
    const { getByTestId } = render(<TabBar />);

    fireEvent.press(getByTestId('tab-asset'));

    expect(mockNavigation.navigate).toHaveBeenCalledWith('ASSET');
  });

  it('should navigate to Liability screen', () => {
    const { getByTestId } = render(<TabBar />);

    fireEvent.press(getByTestId('tab-liability'));

    expect(mockNavigation.navigate).toHaveBeenCalledWith('LIABILITY');
  });
});
