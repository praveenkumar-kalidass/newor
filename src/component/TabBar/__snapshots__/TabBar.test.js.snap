// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppModal should match snapshot 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    onLayout={[Function]}
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Array [
          Object {
            "borderWidth": 0,
            "left": 0,
            "position": "absolute",
            "top": 0,
          },
          Object {
            "height": 1334,
            "width": 750,
          },
        ]
      }
    >
      <View
        dataSet={Object {}}
        style={
          Array [
            Object {
              "backgroundColor": "#ad0000",
              "paddingBottom": 24,
              "paddingLeft": 24,
              "paddingRight": 24,
              "paddingTop": 24,
            },
            Array [
              Object {},
            ],
          ]
        }
      >
        <View
          dataSet={Object {}}
          style={
            Array [
              Object {
                "alignItems": "flex-end",
                "backgroundColor": "transparent",
                "flexDirection": "row",
                "justifyContent": "space-around",
              },
              Array [
                Object {},
              ],
            ]
          }
        >
          <View
            dataSet={Object {}}
            elevation={1}
            shadowColor="#000000"
            shadowOffset={
              Object {
                "height": 1,
                "width": 0,
              }
            }
            shadowOpacity={0.8}
            shadowRadius={1}
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "backgroundColor": "transparent",
                  "position": "absolute",
                  "width": "100%",
                },
                Array [
                  Object {},
                ],
              ]
            }
          >
            <View
              dataSet={Object {}}
              style={
                Array [
                  Object {
                    "backgroundColor": "#eeeeee",
                    "borderRadius": 9999,
                    "height": 96,
                    "marginBottom": 12,
                    "width": 96,
                  },
                  Array [
                    Object {},
                  ],
                ]
              }
            />
          </View>
          <View
            dataSet={Object {}}
            style={Object {}}
          />
          <View
            dataSet={Object {}}
            elevation={1}
            shadowColor="#000000"
            shadowOffset={
              Object {
                "height": 1,
                "width": 0,
              }
            }
            shadowOpacity={0.8}
            shadowRadius={1}
            style={
              Array [
                Object {
                  "backgroundColor": "#eeeeee",
                  "borderRadius": 10,
                  "paddingBottom": 40,
                  "paddingLeft": 40,
                  "paddingRight": 40,
                  "paddingTop": 40,
                  "position": "absolute",
                  "width": "100%",
                },
                Array [
                  Object {},
                ],
              ]
            }
          />
          <View
            dataSet={Object {}}
            style={Object {}}
          />
          <View
            dataSet={Object {}}
            style={
              Object {
                "flex": 1,
                "marginBottom": 8,
                "marginLeft": 16,
              }
            }
          >
            <View
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
              testID="tab-liability"
            >
              <View
                dataSet={Object {}}
                style={
                  Object {
                    "alignItems": "center",
                  }
                }
              >
                <View
                  dataSet={Object {}}
                  style={
                    Object {
                      "alignItems": "center",
                      "borderRadius": 5,
                      "flexDirection": "row",
                      "paddingBottom": 4,
                      "paddingLeft": 4,
                      "paddingRight": 4,
                      "paddingTop": 4,
                    }
                  }
                >
                  <Text
                    allowFontScaling={false}
                    selectable={false}
                    style={
                      Array [
                        Object {
                          "color": "#000000",
                          "fontSize": 30,
                        },
                        undefined,
                        Object {
                          "fontFamily": "FontAwesome",
                          "fontStyle": "normal",
                          "fontWeight": "normal",
                        },
                        Object {},
                      ]
                    }
                  >
                    
                  </Text>
                  <View
                    dataSet={Object {}}
                    style={
                      Object {
                        "width": 4,
                      }
                    }
                  />
                  <Text
                    allowFontScaling={false}
                    selectable={false}
                    style={
                      Array [
                        Object {
                          "color": "#000000",
                          "fontSize": 20,
                        },
                        undefined,
                        Object {
                          "fontFamily": "FontAwesome",
                          "fontStyle": "normal",
                          "fontWeight": "normal",
                        },
                        Object {},
                      ]
                    }
                  >
                    
                  </Text>
                  <View
                    dataSet={Object {}}
                    style={
                      Object {
                        "width": 4,
                      }
                    }
                  />
                  <Text
                    allowFontScaling={false}
                    selectable={false}
                    style={
                      Array [
                        Object {
                          "color": "#000000",
                          "fontSize": 30,
                        },
                        undefined,
                        Object {
                          "fontFamily": "FontAwesome5Free-Solid",
                          "fontStyle": "normal",
                          "fontWeight": "normal",
                        },
                        Object {
                          "fontWeight": "900",
                        },
                      ]
                    }
                  >
                    
                  </Text>
                </View>
                <Text
                  dataSet={Object {}}
                  style={
                    Object {
                      "backgroundColor": undefined,
                      "color": "#171717",
                      "fontFamily": undefined,
                      "fontSize": 16,
                      "fontStyle": "normal",
                      "fontWeight": "700",
                      "letterSpacing": 0,
                      "lineHeight": 24,
                      "textDecorationLine": undefined,
                    }
                  }
                >
                  LIABILITIES
                </Text>
              </View>
            </View>
          </View>
          <View
            dataSet={Object {}}
            style={Object {}}
          />
          <View
            dataSet={Object {}}
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "marginBottom": 4,
              }
            }
          >
            <View
              accessibilityRole="button"
              accessible={true}
              collapsable={false}
              colorScheme="primary"
              dataSet={Object {}}
              focusable={true}
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ad0000",
                  "borderRadius": 9999,
                  "flexDirection": "row",
                  "height": 64,
                  "justifyContent": "center",
                  "width": 64,
                }
              }
            >
              <View
                dataSet={Object {}}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                  }
                }
                test={true}
              >
                <View
                  dataSet={Object {}}
                  style={Object {}}
                >
                  <Text
                    dataSet={Object {}}
                    style={
                      Object {
                        "backgroundColor": undefined,
                        "color": "#fafafa",
                        "fontFamily": undefined,
                        "fontSize": 36,
                        "fontStyle": "normal",
                        "fontWeight": "400",
                        "letterSpacing": 0,
                        "lineHeight": 54,
                        "textDecorationLine": undefined,
                      }
                    }
                  >
                    ₹
                  </Text>
                </View>
              </View>
            </View>
            <Text
              dataSet={Object {}}
              style={
                Object {
                  "backgroundColor": undefined,
                  "color": "#171717",
                  "fontFamily": undefined,
                  "fontSize": 14,
                  "fontStyle": "normal",
                  "fontWeight": "400",
                  "letterSpacing": 0,
                  "lineHeight": 21,
                  "textDecorationLine": undefined,
                }
              }
            >
               
            </Text>
          </View>
          <View
            dataSet={Object {}}
            style={Object {}}
          />
          <View
            dataSet={Object {}}
            style={
              Object {
                "flex": 1,
                "marginBottom": 8,
                "marginRight": 16,
              }
            }
          >
            <View
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
              testID="tab-asset"
            >
              <View
                dataSet={Object {}}
                style={
                  Object {
                    "alignItems": "center",
                  }
                }
              >
                <View
                  dataSet={Object {}}
                  style={
                    Object {
                      "alignItems": "center",
                      "borderRadius": 5,
                      "flexDirection": "row",
                      "paddingBottom": 4,
                      "paddingLeft": 4,
                      "paddingRight": 4,
                      "paddingTop": 4,
                    }
                  }
                >
                  <Text
                    allowFontScaling={false}
                    selectable={false}
                    style={
                      Array [
                        Object {
                          "color": "#000000",
                          "fontSize": 30,
                        },
                        undefined,
                        Object {
                          "fontFamily": "FontAwesome",
                          "fontStyle": "normal",
                          "fontWeight": "normal",
                        },
                        Object {},
                      ]
                    }
                  >
                    
                  </Text>
                  <View
                    dataSet={Object {}}
                    style={
                      Object {
                        "width": 4,
                      }
                    }
                  />
                  <Text
                    allowFontScaling={false}
                    selectable={false}
                    style={
                      Array [
                        Object {
                          "color": "#000000",
                          "fontSize": 20,
                        },
                        undefined,
                        Object {
                          "fontFamily": "FontAwesome",
                          "fontStyle": "normal",
                          "fontWeight": "normal",
                        },
                        Object {},
                      ]
                    }
                  >
                    
                  </Text>
                  <View
                    dataSet={Object {}}
                    style={
                      Object {
                        "width": 4,
                      }
                    }
                  />
                  <Text
                    allowFontScaling={false}
                    selectable={false}
                    style={
                      Array [
                        Object {
                          "color": "#000000",
                          "fontSize": 30,
                        },
                        undefined,
                        Object {
                          "fontFamily": "FontAwesome",
                          "fontStyle": "normal",
                          "fontWeight": "normal",
                        },
                        Object {},
                      ]
                    }
                  >
                    
                  </Text>
                </View>
                <Text
                  dataSet={Object {}}
                  style={
                    Object {
                      "backgroundColor": undefined,
                      "color": "#171717",
                      "fontFamily": undefined,
                      "fontSize": 16,
                      "fontStyle": "normal",
                      "fontWeight": "700",
                      "letterSpacing": 0,
                      "lineHeight": 24,
                      "textDecorationLine": undefined,
                    }
                  }
                >
                  ASSETS
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
      <View
        pointerEvents="none"
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            undefined,
          ]
        }
      />
    </View>
    <View
      elevation={0}
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Array [
          Object {
            "borderWidth": 0,
            "left": -600,
            "position": "absolute",
            "top": 0,
          },
          Object {
            "height": 1334,
            "width": 600,
          },
        ]
      }
    >
      <View
        dataSet={Object {}}
        style={
          Array [
            Object {
              "backgroundColor": "#eeeeee",
              "flex": 1,
              "justifyContent": "space-between",
              "paddingBottom": 80,
              "paddingTop": 80,
            },
            Array [
              Object {},
            ],
          ]
        }
      >
        <View
          dataSet={Object {}}
          style={
            Array [
              Object {
                "borderBottomWidth": 1,
                "borderColor": "#7f7f7f",
                "paddingBottom": 12,
                "paddingLeft": 16,
                "paddingRight": 20,
                "paddingTop": 12,
              },
              Array [
                Object {},
              ],
            ]
          }
        >
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <View
              dataSet={Object {}}
              style={
                Object {
                  "alignItems": "center",
                }
              }
            >
              <View
                dataSet={Object {}}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#a1a1aa",
                    "borderColor": "#27272a",
                    "borderRadius": 9999,
                    "height": 96,
                    "justifyContent": "center",
                    "position": "relative",
                    "width": 96,
                  }
                }
              >
                <Text
                  allowFontScaling={false}
                  selectable={false}
                  style={
                    Array [
                      Object {
                        "color": undefined,
                        "fontSize": 40,
                      },
                      undefined,
                      Object {
                        "fontFamily": "FontAwesome",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
              </View>
              <View
                dataSet={Object {}}
                style={Object {}}
              />
              <Text
                dataSet={Object {}}
                style={
                  Object {
                    "backgroundColor": undefined,
                    "color": "#171717",
                    "fontFamily": undefined,
                    "fontSize": 18,
                    "fontStyle": "normal",
                    "fontWeight": "400",
                    "letterSpacing": 0,
                    "lineHeight": 27,
                    "marginBottom": 8,
                    "marginTop": 8,
                    "textDecorationLine": undefined,
                  }
                }
              >
                undefined undefined
              </Text>
            </View>
          </View>
        </View>
        <RCTScrollView
          contentContainerStyle={
            Array [
              Object {},
              Object {
                "dataSet": Object {},
              },
            ]
          }
          data={
            Array [
              Object {
                "icon": <Icon
                  allowFontScaling={false}
                  color="#000000"
                  name="power-off"
                  size={24}
                />,
                "id": "LOGOUT",
                "label": "LOGOUT",
                "onPress": [Function],
              },
            ]
          }
          dataSet={Object {}}
          getItem={[Function]}
          getItemCount={[Function]}
          keyExtractor={[Function]}
          onContentSizeChange={[Function]}
          onLayout={[Function]}
          onMomentumScrollBegin={[Function]}
          onMomentumScrollEnd={[Function]}
          onScroll={[Function]}
          onScrollBeginDrag={[Function]}
          onScrollEndDrag={[Function]}
          removeClippedSubviews={false}
          renderItem={[Function]}
          scrollEventThrottle={50}
          stickyHeaderIndices={Array []}
          style={Object {}}
          viewabilityConfigCallbackPairs={Array []}
        >
          <View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <View
                accessible={true}
                collapsable={false}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "opacity": 1,
                  }
                }
              >
                <View
                  dataSet={Object {}}
                  style={
                    Array [
                      Object {
                        "borderBottomWidth": 1,
                        "borderColor": "#7f7f7f",
                        "paddingBottom": 12,
                        "paddingLeft": 16,
                        "paddingRight": 20,
                        "paddingTop": 12,
                      },
                      Array [
                        Object {},
                      ],
                    ]
                  }
                >
                  <View
                    dataSet={Object {}}
                    style={
                      Object {
                        "flexDirection": "row",
                      }
                    }
                  >
                    <Text
                      allowFontScaling={false}
                      selectable={false}
                      style={
                        Array [
                          Object {
                            "color": "#000000",
                            "fontSize": 24,
                          },
                          undefined,
                          Object {
                            "fontFamily": "FontAwesome",
                            "fontStyle": "normal",
                            "fontWeight": "normal",
                          },
                          Object {},
                        ]
                      }
                    >
                      
                    </Text>
                    <View
                      dataSet={Object {}}
                      style={
                        Object {
                          "width": 12,
                        }
                      }
                    />
                    <View
                      dataSet={Object {}}
                      style={
                        Object {
                          "justifyContent": "center",
                        }
                      }
                    >
                      <Text
                        dataSet={Object {}}
                        style={
                          Object {
                            "backgroundColor": undefined,
                            "color": "#171717",
                            "fontFamily": undefined,
                            "fontSize": 14,
                            "fontStyle": "normal",
                            "fontWeight": "700",
                            "letterSpacing": 0,
                            "lineHeight": 21,
                            "textDecorationLine": undefined,
                          }
                        }
                      >
                        LOGOUT
                      </Text>
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </RCTScrollView>
      </View>
      <View
        pointerEvents="none"
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            undefined,
          ]
        }
      />
    </View>
  </View>
</RNCSafeAreaProvider>
`;
